### ISSUE TITLE
Move choosing an optimizer from step 6 to step 5 in the deep learning workflow (episodes 1/2)

### ISSUE THREAD

**Issue by colinsauze** [carpentries-incubator/deep-learning-intro#85]
I feel that choosing an optimizer should be part of step 5 (currently "choose a cost function and metrics"), step 6 should just be talking about running the training, deciding on how many epochs, when to stop etc.  It makes more sense to talk about optimizer's alongside loss functions than it does with running the training.  Seeing as Keras combines choosing the optimizer with the loss function in the compile statement, it would make sense to talk about these together. 

My latest pull request for episode 1 (pr #84) has it this way, but I can change it back if others don't think this is a good idea.

- [ ] Move the compile line to step 5 
- [ ] Move explanation of choosing a optimizer to step 5

**Comment by bpmweel** [comment]
- [ ] Move the compile line to step 5 
- [ ] Move explanation of choosing a optimizer to step 5

**Comment by colinsauze** [comment]
Just revisiting this, the pull request was accepted already but that only changes episode 1. Further changes are needed in episode 2.
